db/migrate/20160203195736_remove_docker_image_schema.rb

```diff
@@ -0,0 +1,39 @@
+class RemoveDockerImageSchema < ActiveRecord::Migration
```

Comment:
@beav, Can the upgrade script in https://github.com/Katello/katello/pull/5744 be run before the db:migrate is executed?  I raised the question on that PR and I got the impression that it could.  I assumed 'entirely malleable' meant we could control when it is run.